scargo/__init__.py

```diff
@@ -7,11 +7,9 @@
 
 from typer import Argument, Option, Typer
 
-from scargo.scargo_src.global_values import (
-    DESCRIPTION,
-    SCARGO_DEFAULT_CONFIG_FILE,
-    __version__,
-)
+__version__ = "1.0.2"
```

Comment:
this will break version usage elsvare

Reply:
Everything is working (look at tests), everything is adjusted according to this.